spring-cloud-dataflow-server-core/src/main/resources/application-init-mariadb.yml

```diff
@@ -15,4 +15,6 @@ spring:
         - classpath*:/schemas/mariadb/V6-dataflow.sql
         - classpath*:/schemas/mariadb/V7-dataflow.sql
         - classpath*:/schemas/mariadb/V9-dataflow.sql
-        - classpath*:/schemas/mariadb/V8-dataflow.sql
\ No newline at end of file
+        - classpath*:/schemas/mariadb/V8-dataflow.sql
```

Comment:
Looks like v9 is before v10 although v9 was before v8 already??

Reply:
It was like this before the introduction of this PR.

Reply:
Agreed. But was that intentional?

Reply:
I'll investigate and remediate if necessary